content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -729,6 +730,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `ServiceAppProtocol`: Enables the `AppProtocol` field on Services and Endpoints.
 - `ServiceLBNodePortControl`: Enables the `spec.allocateLoadBalancerNodePorts`
   field on Services.
+- `ServiceLoadBalancerClass`: Enables the `LoadBalancerClass` field on Services.
```

Comment:
Do we have any plan to further document this new field?

Reply:
It does feel like we should.

Reply:
Added KEP link for this field, thanks😊